src/chfn.c

```diff
@@ -620,10 +620,12 @@ int main (int argc, char **argv)
 	char                 *user;
 	const struct passwd  *pw;
 
+	sanitize_env ();
+	check_fds ();
+
 	log_set_progname(Prog);
 	log_set_logfd(stderr);
 
```

Comment:
The commit message should mention sanitize_env().

Reply:
The call to sanitize_env was already being called, I just moved it up to be the same as all other programs. I can add a note about it but I'm not sure what I would say.

Reply:
Ahh, I see, you've only moved it, never added it.  Then forget what I said.  :)